cloudfoundry-client-reactor/src/main/java/org/cloudfoundry/reactor/util/UriQueryParameters.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.reactor.util;
+
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+public class UriQueryParameters {
+
+    public static void set(UriComponentsBuilder builder, Stream<UriQueryParameter> uriQueryParameters) {
+        // Replace all literal values with URI variables to apply more strict encoding:
```

Comment:
We rarely include comments as it's often a sign that the code itself is obscure, or a method within it could be named better. Having said that I don't see anything here that should obviously be changed, but if you have any ideas feel free!

Reply:
I also prefer to not include comments, but I think this one will be helpful, since it's not immediately obvious why we're replacing literal values with URI variables.